FunctionalTableDataTests/TableSectionChangeSetTests.swift

```diff
@@ -63,6 +63,31 @@ class TableSectionChangeSetTests: XCTestCase {
 		XCTAssertEqual(changes.deletedSections, IndexSet())
 	}
 
+	func testSectionEquality() {
```

Comment:
Be more expressive here `testNothingIsDiffedWithEqualSections` or something like that